include/seqan3/io/sequence_file/format_genbank.hpp

```diff
@@ -0,0 +1,266 @@
+        ++stream_it;
+        ++stream_it;
+        ++stream_it;
+    }
+
+    //!\copydoc SequenceFileOutputFormat::write
+    template <typename stream_type,     // constraints checked by file
+              typename seq_type,        // other constraints checked inside function
+              typename id_type,
+              typename qual_type>
+    void write(stream_type                        & stream,
+               sequence_file_output_options const & options,
+               seq_type                           && sequence,
+               id_type                            && id,
+               qual_type                          && SEQAN3_DOXYGEN_ONLY(qualities))
+    {
+        std::ranges::ostreambuf_iterator stream_it{stream};
+        size_t sequence_size = 0;
+        if constexpr (!detail::decays_to_ignore_v<seq_type>)
+            sequence_size = ranges::size(sequence);
+
+        // ID
+        if constexpr (detail::decays_to_ignore_v<id_type>)
+        {
+            throw std::logic_error{"The ID field may not be set to ignore when writing genbank files."};
+        }
+        else if (ranges::empty(id)) //[[unlikely]]
+        {
+            throw std::runtime_error{"The ID field may not be empty when writing genbank files."};
+        }
+        else if (options.embl_genbank_complete_header)
+        {
+            std::ranges::copy(id, stream_it);
+        }
+        else
+        {
+            std::ranges::copy(std::string_view{"LOCUS       "}, stream_it);
+            std::ranges::copy(id, stream_it);
+            std::ranges::copy(std::string_view{"                 "}, stream_it);
+            std::ranges::copy(std::to_string(sequence_size), stream_it);
+            std::ranges::copy(std::string_view{" bp\n"}, stream_it);
+        }
+
+        // Sequence
+        if constexpr (detail::decays_to_ignore_v<seq_type>) // sequence
+        {
+            throw std::logic_error{"The SEQ field may not be set to ignore when writing genbank files."};
+        }
+        else if (ranges::empty(sequence)) //[[unlikely]]
+        {
+            throw std::runtime_error{"The SEQ field may not be empty when writing genbank files."};
+        }
+        else
+        {
+            std::ranges::copy(std::string_view{"ORIGIN"}, stream_it);
+            stream_it = '\n';
+            auto seq = sequence | ranges::view::chunk(60);
+            unsigned int i = 0;
+            size_t bp = 1;
+            while (bp < sequence_size)
+            {
+                for(size_t j = std::to_string(bp).length(); j < 9; j++)
+                    stream_it = ' ';
+                std::ranges::copy(std::to_string(bp), stream_it);
+                stream_it = ' ';
+                std::ranges::copy(seq[i] | view::to_char
+                                         | ranges::view::chunk(10)
+                                         | std::view::join(' '), stream_it);
+                bp += 60;
+                ++i;
+                stream_it = '\n';
```

Comment:
There is something like
```suggestion
               detail::write_eol();
```
That should be used for writing endline characters because then we can easily adapt to windows at some point